filebeat-yml-script.py

```diff
@@ -145,5 +169,10 @@ def _include_containers():
 else:
     _exclude_containers()
 
+if "excludeLines" in os.environ:
+    _exclude_lines()
+
+if "includeLines" in os.environ:
+    _include_lines()
```

Comment:
@jeremyj According to your comment we should change the order of exclude/include here